pkg/build/build.go

```diff
@@ -93,6 +94,8 @@ type KernelError struct {
 	Report      []byte
 	Output      []byte
 	Maintainers []string
+	Cc          []string
```

Comment:
Why do we need CC here?
I would assume Recipients should eliminate the need in both CC and Maintainers.